src/vs/workbench/browser/parts/views/viewPane.ts

```diff
@@ -579,7 +579,7 @@ export abstract class ViewPane extends Pane implements IView {
 						if (typeof node === 'string') {
 							append(p, document.createTextNode(node));
 						} else {
-							const link = this.instantiationService.createInstance(Link, node);
+							const link = this.instantiationService.createInstance(Link, node, {});
```

Comment:
The last parameter is optional, does the instantiation service make you pass it in?

Reply:
TS doesn't allow the required service injection parameters follow an optional parameter.